hedera-node/hedera-app/build.gradle.kts

```diff
@@ -52,13 +52,15 @@ dependencies {
         itestImplementation(gav("com.swirlds.common"))
         itestImplementation(gav("com.swirlds.config"))
         itestImplementation(gav("io.grpc"))
-        itestImplementation(gav("io.helidon.grpc.client"))
-        itestImplementation(gav("io.helidon.grpc.server"))
         itestImplementation(gav("org.apache.logging.log4j"))
         itestImplementation(gav("org.assertj.core"))
         itestImplementation(gav("org.bouncycastle.provider"))
         itestImplementation(gav("org.junit.jupiter.api"))
         itestImplementation(gav("org.junit.jupiter.params"))
+        itestImplementation(gav("io.netty.transport.classes.epoll"))
+        itestImplementation(gav("io.netty.transport.epoll"))
+        itestImplementation("io.netty:netty-transport-native-epoll:4.1.94.Final")
```

Comment:
This is different from what GRPC requires (4.1.87.Final).  We probably want to make sure the versions match to avoid more split-package issues.
Also, most versions are managed in platform now, so might be good to cross check that this matches that version (or better, manage the version there).

Reply:
Agreed, this is the same version in the BOM. I'm not sure why I had to call it out, but it wouldn't build otherwise. I could use @jjohannes's help here.

Reply:
This should not be needed.

As far as I can see, if I remove the following two lines, everything still works fine:

```
itestImplementation("io.netty:netty-transport-native-epoll:4.1.94.Final")
itestImplementation("io.netty:netty-transport-classes-epoll:4.1.94.Final")
```

What's see issue you have been seeing?